Creational/SimpleFactory/Usage/SimpleFactoryUsage.php

```diff
@@ -0,0 +1,9 @@
+<?php
```

Comment:
Isn't there a unit test that shows the usage of the SimpleFactory? This file serves no real purpose, please remove it.